Amazon.AspNetCore.DataProtection.SSM/Amazon.AspNetCore.DataProtection.SSM.csproj

```diff
@@ -2,6 +2,13 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <NetStandardImplicitPackageVersion>2.0.0</NetStandardImplicitPackageVersion>
```

Comment:
You are missing the following:
GeneratePackageOnBuild, Authors, Copyright, PackageProjectUrl, PackageLicenseUrl, PackageIconUrl, RepositoryUrl, Company

Reply:
See: https://github.com/aws/aws-aspnet-cognito-identity-provider/blob/master/src/Amazon.AspNetCore.Identity.Cognito/Amazon.AspNetCore.Identity.Cognito.csproj

Reply:
added